Makefile.toml

```diff
@@ -0,0 +1,154 @@
+
+######################
+#
+# public tasks:
+# * stable-checks
+# * run-tests
+#
+######################
+
+[config]
+min_version = "0.32.2"
+default_to_workspace = false
+
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+CARGO_MAKE_CLIPPY_ARGS = "-- --deny=warnings"
+CARGO_MAKE_CARGO_BUILD_TEST_FLAGS = ""
+
+[config.modify_core_tasks]
+private = true
+namespace = "core"
+
+[tasks.stable-checks]
+description = "Runs all validations for both workspaces on rust stable only."
+condition = { channels = [ "stable" ] }
+run_task = { name = [ "stable-checks-flow", "stable-checks-stdweb" ], fork = true }
+
+[tasks.run-tests]
+dependencies = ["tests-setup"]
+env = { CARGO_MAKE_WORKSPACE_SKIP_MEMBERS = ["examples/*", "yewtil/examples/*"] }
+run_task = { name = [ "run-tests-flow", "run-tests-stdweb", "tests-cleanup" ], fork = true }
```

Comment:
The "tests-cleanup" task a prior task fails. Is there something similar to `on_error_task` that always runs the task regardless of whether there was an error or not?

Reply:
let me check. i could use on error task but the setup is so complex as it is... so let me check it to make sure.